test/unit/driver-etcd.js

```diff
@@ -20,12 +21,19 @@
 
 
     var s1 = new Server();
+    var serverTemp = './tmp/etcd-test';
```

Comment:
Does Node provide a function to create/get temporary directories, no matter the OS?